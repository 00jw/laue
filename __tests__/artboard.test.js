const {Artboard} = require('../dist/laue.umd')
const {mount} = require('@vue/test-utils')

describe('Object:Artboard', () => {
  it(':data', () => {
    const wrapper = mount(Artboard)
    console.log(wrapper)
  })

  it('matches snapshot', () => {
    const wrapper = mount(Artboard)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
