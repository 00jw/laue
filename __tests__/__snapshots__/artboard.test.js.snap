// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Object:Artboard matches snapshot 1`] = `
<div style="position: relative; width: 300px;">
  <svg width="300" height="75" viewBox="0 0 300 75"></svg>
</div>
`;
